balena.yml

```
@@ -0,0 +1,20 @@
+
+name: leviathan
+type: sw.application
+description: >-
+  Leviathan is a distributed testing framework. 
+assets:
+  repository:
+    type: blob.asset
+    data:
+      url: 'https://github.com/balena-os/leviathan'
+data:
+  applicationEnvironmentVariables:
+    - TESTBOT_DUT_TYPE:
+    - SCREEN_CAPTURE:
+    - TESTBOT_PSU_ADDRESS:
+    - BALENA_HOST_CONFIG_dtoverlay: "balena-fin","uart1,txd1_pin=32,rxd1_pin=33"
```

```suggestion
    - BALENA_HOST_CONFIG_dtoverlay: >
      "balena-fin","uart1,txd1_pin=32,rxd1_pin=33"
```
I think a scalar may avoid the need to escape the double quotes. There are other workarounds as well.